readthedocs/doc_builder/director.py

```diff
@@ -331,12 +322,17 @@ def install(self):
 
     # Build
     def build_html(self):
+        if self.run_build_job("build.html"):
+            return True
```

Comment:
I don't like this pattern where we are running all the commands in an `if` conditional. I'm proposing an easier to read and simpler implementation for this in my previous comment.

Reply:
>  From what I can tell, it only returns None?

It returns true if the command was executed, false otherwise.

I can re-write this as

```python
if self.data.config.build.jobs.html is not None:
    return self.run_build_job("build.html")
```

but feels like we are duplicating things